### DISCUSSION TITLE
Add option to show only button on service call panel

### DISCUSSION THREAD

**Discussion by Scout22** [foxglove/community#894] [Q&A] [answered]
Topic publisher panel has an option to hide the message and just display a button that can be clicked.
It would be nice if the same could be done for the service panel.
Where we would just define a predetermine service call then hide everything and leave just the button to allow a cleaner dashboard

**Comment by achim-k** [comment, marked as answer]
This feature has been implemented and the new `Editing mode` panel setting is already available at https://app.foxglove.dev. 

![image](https://github.com/foxglove/community/assets/9250155/6b12bfd0-28bf-462b-a036-59354e9e1850)
